static/chats.js

```diff
@@ -163,6 +169,9 @@ function paradiPalidzibu() {
   return 'Pieejamās komandas : "/vards JaunaisVards", "/palidziba", "/versija"'
 }
 
+function dzestZinju() {
```

Comment:
Šo vajag?

Reply:
Sāku darīt 2 lietas vienlaicīgi un vispirms ieliku kodā to, ko protu izdarīt. No PR šo var ņemt ārā. Man pašai lokāli ir, ceru turpināt.